src/NuGet.Core/NuGet.Protocol/LegacyFeed/V2FeedParser.cs

```diff
@@ -43,6 +43,7 @@ public sealed class V2FeedParser : IV2FeedParser
         private static readonly XName _xnameLicenseUrl = XName.Get("LicenseUrl", DataServicesNS);
         private static readonly XName _xnameProjectUrl = XName.Get("ProjectUrl", DataServicesNS);
         private static readonly XName _xnameTags = XName.Get("Tags", DataServicesNS);
+        private static readonly XName _xnameGalleryDetailsUrl = XName.Get("GalleryDetailsUrl", DataServicesNS);
```

Comment:
Has this already been implemented in NuGetGallery? Can we use the name `PackageDetailsUrl`, as is used in other classes in your change? Or `PackageDetailsUrlTemplate` or `PackageDetailsBaseUrl`, whatever's appropriate?

Reply:
Already implemented in Gallery. See:
https://www.nuget.org/api/v2/Packages(Id='Newtonsoft.Json',Version='9.0.1')

I would rather not propagate the "gallery" prefix to the V3 protocol that's a bit too specific.

Reply:
i think the gallery prefix is also too specific for the v2 protocol, but it's probably not worth changing since it's already implemented.